lists/gb/gb-edu.json

```diff
@@ -1,18 +1,17 @@
 {
   "name": {
     "en": "Register of Schools (England and Wales)",
-    "local": "'Edubase', UK Register of Learning Providers"
+    "local": "Get information about schools, UK Register of Learning Providers"
   },
   "url": "https://get-information-schools.service.gov.uk",
   "description": {
-    "en": "Schools and Colleges in England must be registered with the Department of Education, and in Wales with the Welsh Government.[1]\n\nThe Register of Schools is maintained by the Department of Education and provides a URN for each school, university and other educational establishment in England and Wales.\n\nThe full Register of Schools in England is available (Alpha version) on https://registers.cloudapps.digital/registers?phase=in+progress. **The register for Welsh Schools is not yet launched and so all schools in Wales may not yet be present on this list.**\n\n[1] This includes independent schools which meet the criteria on https://www.gov.uk/independent-school-registration"
+    "en": "Schools and Colleges in England must be registered with the Department of Education.[1]\n\nThe Register of Schools is maintained by the Department of Education and provides a URN for each school, university and other educational establishment in England.\n\nThe full Register of Schools in England is available at https://get-information-schools.service.gov.uk/Downloads.\n\n **The register for Welsh Schools used to be part of this list, but is now the responsibility of the Welsh government. Details of the welsh list are available on org-id.guide.**\n\n[1] This includes independent schools which meet the criteria on https://www.gov.uk/independent-school-registration\n\n[2]http://org-id.guide/list/GB-WALEDU"

```

Comment:
Capitalisation not quite right?

"The register for Welsh **S**chools used to be part of this list, but is now the responsibility of the Welsh government. Details of the **w**elsh list are ..."

I'd put:

"The register for Welsh schools used to be part of this list, but is now the responsibility of the Welsh government. Details of the Welsh list are ..."

Reply:
Resolved with https://github.com/org-id/register/pull/372/commits/de2ebe15ee672de5cee19a9d270499ad9c6261b9